tools/indexer/example/src/main.rs

```diff
@@ -176,15 +176,16 @@ async fn listen_blocks(mut stream: mpsc::Receiver<near_indexer::BlockResponse>)
         //         ),
         //     ],
         // }
-        eprintln!("{:#?}", block);
+        // eprintln!("{:#?}", block);
+        eprintln!("Example: block height {:?}", block.block.header.height);
     }
 }
 
 fn main() {
     // We use it to automatically search the for root certificates to perform HTTPS calls
     // (sending telemetry and downloading genesis)
     openssl_probe::init_ssl_cert_env_vars();
-    init_logging(true);
+    init_logging(false);
```

Comment:
Meh. Let's remove that `verbose` flag completely and only leave the `false` branch

Reply:
How do you like the idea if I rewrite `init_logging` function to look like https://github.com/near/near-indexer-for-wallet/blob/master/src/main.rs#L195-L201 ?

Reply:
Sounds good to me